src/main/ts/core/BranchLogic.ts

```diff
@@ -18,12 +20,21 @@ export const getReleaseBranchName = ({ major, minor }: MajorMinorVersion): strin
 
 // eslint-disable-next-line no-shadow
 export const enum BranchType {
-  Main, Feature, Hotfix, Spike, Release
+  main = 'main',
```

Comment:
How come you renamed all these enums? Most times I see them used they are PascalCase or all uppercase (since they are effectively constants).

Reply:
I put a comment in the commit message. I was trying to make things more consistent, so the `main` branch is branch type `main` in branch state `main` with JSON output `main`. Happy to discuss this, though.

Reply:
As discussed out-of-band, we decided to use TitleCase to be consistent with TinyMCE and the TypeScript handbook. Done.